### ISSUE TITLE
[Feature Request] Raids

### ISSUE THREAD

**Issue by lakafior** [fgl27/SmartTwitchTV#185]
**Please describe the feature.**
Usually streamer in the end of the stream is raiding to someone else. Right now the app is not recognizing that and just ends the stream. It would be great to follow the raid and switch to a raided stream.

**Additional context or samples**
https://help.twitch.tv/s/article/how-to-use-raids?language=en_US
https://dev.twitch.tv/docs/api/raids/

**Comment by nathanchere** [comment]
I consider not being sent to someone else's stream a feature already and is one of the major selling points of this over the stock Twitch app for me. If this 'feature' is added, I would greatly prefer the option to disable it.

**Comment by NightHammer1000** [comment]
Yeah. It would be greatly appreciated if we could turn it off if that feature makes it in.

Nothing more annoying falling asleep to a chill stream only to get sent to some Hyperactive Dude screaming and get rudely awoken.

**Comment by housemix** [comment]
Hello !

Thank you for developing such a good app !

Any update on raids? Will it happen?

Thank you again!

**Comment by housemix** [comment]
No idea I'm completely clueless but I'll see what I can do ahah.
Sorry if I made a wrong question or something, I'm not very familiar with this
